packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/create-an-okta-application/index.md

```diff
@@ -3,6 +3,6 @@ title: Create an Okta application
 ---
 Before you can sign a user in, you need to create an Okta application that represents your mobile application.
 
-Use the Okta application that you created when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/create-okta-application/) guide.
+Use the Okta application that you created when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/main/) guide.
```

Comment:
```suggestion
Use the Okta application that you created when you walked through the [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/) guide.
```
Removing "/main/" is cleaner. Plus, it avoids the broken nav and bread-crumbs bug.